src/vs/workbench/parts/tasks/node/processTaskSystem.ts

```diff
@@ -105,6 +105,10 @@ export class ProcessTaskSystem implements ITaskSystem {
 		return true;
 	}
 
+	public extensionCallbackTaskComplete(task: Task): Promise<void> {
+		return Promise.resolve();
```

Comment:
This should really never happen since executeTask bails out for anything but CustomTask. Can you add a notification here so that we can see if we ever get into this state?